client/src/lib/components/EmailCreator/EmailCreator.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+  export let contactEmails: string[];
+  let contactEmailErrors: string[] = [];
+
+  const handleChange = (
+    index: number,
+    event: Event & {
+      currentTarget: EventTarget & HTMLInputElement;
+    }
```

Comment:
Est-ce que ça marche aussi ?

```suggestion
    event: InputEvent
```